common/src/main/java/com/wynntils/core/consumers/overlays/Overlay.java

```diff
@@ -63,6 +76,35 @@ protected Overlay(
         this.verticalAlignmentOverride.store(verticalAlignmentOverride);
     }
 
+    /**
+     * <li>The default condition that determines whether an overlay is going to be rendered.</li>
+     * <li>Overwrite this to completely change the condition, or overwrite {@link #additionalRenderCondition()} instead to only add to the default condition.</li>
+     * <li>By default, overlays only render when on world and not riding a Display entity</li>
+     */
+    protected boolean defaultRenderCondition() {
+        return Models.WorldState.onWorld()
+                && (McUtils.player() != null && !(McUtils.player().getVehicle() instanceof Display));
+    }
+
+    /**
+     * <li>Additional condition that is added onto {@link #defaultRenderCondition() default condition}, both of them need to be true</li>
+     * <li>Overwrite this to add to the default or overwrite {@link #defaultRenderCondition()} to completely change the condition</li>
+     * <li>By default, it is always true</li>
+     */
+    protected boolean additionalRenderCondition() {
+        return true;
+    }
```

Comment:
Why two methods? If I want additional conditions, I override and call super and-ed with my other conditions.

Reply:
Okay, I get that. This just demonstrates that I also don't like the current name and my brain automatically wanted to just get rid of this all together. I overall feel like that we are so overwhelming the simple enabled / disabled concept that it is getting hard for even us to understand. Can we spend a bit of time thinking through what we really need?